WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -2334,6 +2336,7 @@ k_republic = {
 		prestige = 200
 		independent = yes
 		realm_size = 50
+		has_global_flag = senate_not_active
```

Comment:
What happened here?

Reply:
I wanted to prevent both k_republic and d_senate existing at the same time, but I will come with better solution to this.

Reply:
But k_republic is the successor title of d_senate. It is there for the ruler of d_senate.

Reply:
But what is the point of rulling both titles? It's like keeping the ERE title after reforming RE.
k_republic creator should lose d_senate and the title history should be copied.

Reply:
That I can understand, but s you said this particular change is counter-productive if that is the aim.

Reply:
I think it is an excellent idea!

Reply:
The decision has been added, can be reviewed.

Reply:
I left one comment, not much to say.